src/clib/old_tests/res_util/test_ert_util_subst_list_add_from_string.cpp

```diff
@@ -32,6 +33,21 @@ void test_invalid_arg_string(const char *arg_string) {
     free(tmp);
 }
```

Comment:
I suppose `test_invalid_arg_string` and `call_func_util_abort` can now be removed?

Reply:
Good catch!